libs/platform/ebpf_crypto_hash.c

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "ebpf_platform.h"
+
+#include <bcrypt.h>
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md says:
"**DO** include system headers (with `<>`) before local headers (with `""`), and list them in alphabetical order where possible."

Reply:
Fixed.